packages/contracts/contracts/discovery/Controller.sol

```diff
@@ -67,6 +67,20 @@ contract Controller is IController, AccessControl {
         projects[address(project)] = true;
     }
 
+    /// @inheritdoc IController
+    function approveByOwner(address _project)
+        external
+        override(IController)
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        require(
+            IProject(_project).approvedByOwner() == false,
+            "already approved"
+        );
+
+        IProject(_project).approveByOwner();
+    }
```

Comment:
wait, I'm confused:
1. don't we already have `approveByManager` on the project?
2. didn't we say this one was going to be done directly via the project, and not via the controller (which again, is what `approveByManager` is doing)?

Reply:
sorry, I didn't realize during the morning that we were just duplicating work :grimacing: context switchign is hard

Reply:
don't worry, I probably also didn't make that good of a job explaining why I was doing it